Makefile

```diff
@@ -21,7 +21,7 @@
 ###
 
 # Default shell `/bin/sh` has different meanings depending on the platform.
-SHELL := /bin/bash
+SHELL := $(shell command -v bash)
```

Comment:
I don't know much about Makefiles but it seems like just `SHELL := bash` works which seems simpler

Reply:
Good point - I can't immediately think of any downside of just using `SHELL += bash` other than possibly a little overhead from search $PATH more than once. I was just re-purposing an earlier freebsd-only fix which used `$(shell command ...)`.

Reply:
One difference is that `command -v` yields a full absolute path, as opposed to just `bash`. I don't know how `$SHELL` cascades, or if a full path is important, but it's worth keeping in mind.